test-requirements.txt

```diff
@@ -1 +1,3 @@
 odoo_test_helper
+opencv-python
```

Comment:
Hello sir, fyi `opencv-python-headless` is [more suitable for servers](https://github.com/opencv/opencv-python#installation-and-usage) (doesn't require `libgl1` deb package)

![image](https://github.com/OCA/server-ux/assets/511258/0ffc6cbf-96dd-41e5-b847-4f83db1d53e7)

Reply:
I see, I didn't know, thanks for that :smile: